gapic/src/main/com/google/gapid/util/Colors.java

```diff
@@ -42,6 +44,18 @@ public static RGB rgb(double r, double g, double b) {
     return new RGB((int)(r * 255), (int)(g * 255), (int)(b * 255));
   }
 
+  public static RGB getRandomColor(int index) {
```

Comment:
The colors were not supposed to be random. The code attempts to generate complementary colors for close indices, while also avoiding duplicates as the number of indices grow.